src/main/java/com/vaadin/demo/component/datepicker/DatePickerInternationalization.java

```diff
@@ -0,0 +1,33 @@
+package com.vaadin.demo.component.datepicker;
+
+import java.util.List;
+
+import com.vaadin.demo.DemoExporter;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.router.Route;
+
+@Route("date-picker-internationalization")
+public class DatePickerInternationalization extends Div {
+
+    public DatePickerInternationalization() {
+        DatePicker datePicker = new DatePicker("Sitzungsdatum");
+
+        // tag::snippet[]
+        DatePicker.DatePickerI18n customI18n = new DatePicker.DatePickerI18n();
```

Comment:
```suggestion
        DatePicker.DatePickerI18n germanI18n = new DatePicker.DatePickerI18n();
```
Makes it less confusing what kind of language is actually being used.

Reply:
I changed the object name to "germanI18n".